MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewController.swift

```diff
@@ -8,10 +8,25 @@
 import UIKit
 
 class MovieDetailsViewController: UIViewController {
-    private let movieDetailsViewModel = MovieDetailsViewModel()
 
+    // MARK: - IBOutlets
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var posterImageView: UIImageView!
+
+    // MARK: - Variables
+    var viewModel = MovieDetailsViewModel()
+    private var selectedSearchMovie: SearchMoviesResults?
+    private var selectedHomeScreenMovies: NowPlayingResults?
+
+    // MARK: - Functions
     override func viewDidLoad() {
         super.viewDidLoad()
-        movieDetailsViewModel.fetchMovieDetails()
+        viewModel.displayMovieDetails { [weak self] title, posterPath in
```

Comment:
this should be done in the viewModel. Please move it there like you did for the other files below

Reply:
understood